android/local.properties

```diff
@@ -1,5 +1,5 @@
-sdk.dir=C:\\Users\\john\\AppData\\Local\\Android\\sdk
-flutter.sdk=C:\\src\\flutter
+sdk.dir=/home/timothy/Android/Sdk
```

Comment:
Should add this file to gitignore and delete

Reply:
Removed